DI/ValidatorExtension.php

```diff
@@ -29,9 +29,14 @@ public function loadConfiguration()
 		if (!$builder->parameters['debugMode']) {
 			return;
 		}
-
+		
+		$presentersToDisableValidationFor = array();
+		if(isset($this->config['presentersToDisableValidationFor'])) {
+			$presentersToDisableValidationFor = is_array($this->config['presentersToDisableValidationFor']) ? $this->config['presentersToDisableValidationFor'] : array($this->config['presentersToDisableValidationFor']);
```

Comment:
There is condition which determinate if config parameter is single string and convert it to array. If you think is unnecessary I can change it like this:

```php
if(isset($this->config['presentersToDisableValidationFor']) && is_array($this->config['presentersToDisableValidationFor'])) {
  $presentersToDisableValidationFor =$this->config['presentersToDisableValidationFor'];
}
```

Reply:
Ah I see. Yup, I'd drop it, the array is just fine and the complexity is lower then